src/huggingface_hub/inference/_client.py

```diff
@@ -95,6 +96,14 @@
 logger = logging.getLogger(__name__)
 
 
+# Add dataclass for ModelStatus. We use this dataclass in get_model_status function.
+@dataclass
+class ModelStatus:
+    loaded: bool
+    state: str
+    compute_type: str
+    framework: str
```

Comment:
Would be nice to add a docstring here. This is important as it is used to generate our documentation pages. [Here is an example](https://github.com/huggingface/huggingface_hub/blob/4f1e4bbb28b58ffbc2588d2c549c6be0d083bac8/src/huggingface_hub/_space_api.py#L67) of documented dataclass.

Reply:
Got you :+1: I have already a doc string in the function that I implemented `get_model_status` I will do it to `dataclass` too.